doc/user/content/ingest-data/cdc-sql-server.md

```diff
@@ -0,0 +1,172 @@
+
+## Kafka + Debezium
+
+Utilize [Debezium](https://debezium.io/) and the [Kafka source](/sql/create-source/kafka/#using-debezium) to propagate CDC data from SQL Server to Materialize. Debezium captures row-level changes resulting from `INSERT`, `UPDATE`, and `DELETE` operations in the upstream database and publishes them as events to Kafka using Kafka Connect-compatible connectors.
+
+### Database Setup
+
+Before deploying a Debezium connector, ensure that the upstream database is configured to support CDC.
+
+1.  Enable CDC on the SQL Server instance and database:
+
+    ```sql
+    -- Enable CDC on the SQL Server instance
+    USE MyDB
+    GO
+    EXEC sys.sp_cdc_enable_db
+    GO
+    ```
+
+### Deploy Debezium
+
+**Minimum requirements:** Debezium 1.5+
+
+Debezium is deployed as a set of Kafka Connect-compatible connectors. First, define a SQL Server connector configuration, then start the connector by adding it to Kafka Connect.
+
+{{< tabs >}}
+{{< tab "Debezium 1.5+">}}
+
+1.  Create a connector configuration file and save it as `register-sqlserver.json`:
+
+    ```json
+    {
+        "name": "inventory-connector",
+        "config": {
+            "connector.class": "io.debezium.connector.sqlserver.SqlServerConnector",
+            "tasks.max": "1",
+            "database.server.name": "server1",
+            "database.hostname": "sqlserver",
+            "database.port": "1433",
+            "database.user": "sa",
+            "database.password": "Password!",
+            "database.names": "testDB",
+            "database.history.kafka.bootstrap.servers": "<broker>:9092",
+            "database.history.kafka.topic": "schema-changes.inventory",
+            "key.converter": "io.confluent.connect.avro.AvroConverter",
+            "value.converter": "io.confluent.connect.avro.AvroConverter",
+            "key.converter.schema.registry.url": "http://<schema-registry>:8081",
+            "value.converter.schema.registry.url": "http://<schema-registry>:8081",
+            "database.encrypt": "false"
+        }
+    }
+    ```
+
+    You can read more about each configuration property in the [Debezium documentation](https://debezium.io/documentation/reference/1.9/connectors/sqlserver.html).
+
+{{< /tab >}}
+{{< tab "Debezium 2.0+">}}
+
+1. Beginning with Debezium 2.0.0, Confluent Schema Registry support is not included in the Debezium containers. To enable the Confluent Schema Registry for a Debezium container, install the following Confluent Avro converter JAR files into the Connect plugin directory:
+    * `kafka-connect-avro-converter`
+    * `kafka-connect-avro-data`
+    * `kafka-avro-serializer`
+    * `kafka-schema-serializer`
+    * `kafka-schema-registry-client`
+    * `common-config`
+    * `common-utils`
+
+1.  Create a connector configuration file and save it as `register-sqlserver.json`:
```

Comment:
Same as `sqlserver.json`. Maybe I'm missing something.

Reply:
Hm I am not 100% sure what you mean here?

Reply:
It was the same as the other comment, but probably you have the same answer about crucial config missing! So nothing to do here.

Reply:
Ah I see! Thank you for the clarification.